android/src/main/kotlin/com/adyen/checkout/flutter/cse/AdyenCSE.kt

```diff
@@ -0,0 +1,38 @@
+package com.adyen.checkout.flutter.cse
+
+import EncryptedCardDTO
+import UnencryptedCardDTO
+import com.adyen.checkout.cse.CardEncrypter
+import com.adyen.checkout.flutter.utils.ConfigurationMapper.fromDTO
+import com.adyen.checkout.flutter.utils.ConfigurationMapper.mapToEncryptedCardDTO
+import java.lang.Exception
+
+class AdyenCSE {
```

Comment:
This is intended to be public right?

Reply:
Not really, let's make it internal. There is no need to expose it publicly.